src/pages/proposal.tsx

```diff
@@ -197,9 +198,12 @@ export default function ProposalPage() {
     isMobile
   )
 
+  const [renderFloatingBar, setRenderFloatingBar] = useState<boolean>(false)
+  const [isFloatingHeaderVisible, setIsFloatingHeaderVisible] = useState<boolean>(true)
```

Comment:
What is the different between Floating Bar and Floating Header? Can't these be both the same condition?